src/components/CategorySelector.tsx

```diff
@@ -93,19 +100,45 @@ export const CategorySelector = ({
           InputProps={{ endAdornment: <SearchIcon /> }}
           onChange={(event) => setSearchValue(event.target.value)}
         />
-        <Box
-          sx={{
-            display: 'flex',
-            gap: '0.5rem',
-            alignItems: 'center',
-          }}>
-          <FilterVintageIcon
+        {canSelectAllNviCategories ? (
```

Comment:
Kan man ha `canSelectAllNviCategories` uten `setSelectedCategories`, og vice-versa?
Lurer på om `canSelectAllNviCategories` nå egentlig bare har som ansvar å sette en prop for om `setSelectedCategories` sendes med 🤔 
I så fall tror jeg `setSelectedCategories` egentlig er en (optinal) metode `selectAllNviCategories` elns.
Det nye her er vel at enten kan man velge alle NVI-typer, eller så skal man ikke. Da tror jeg en ny prop dekker det

```suggestion
        {selectAllNviCategories ? (
```

Reply:
Brukte `setSelectedCategories`